Docs/articles/Kernel/Network.md

```diff
@@ -49,6 +49,25 @@ using(var xClient = new UdpClient(4242))
 }
 ```
 
+## TCP
+Like UDP, TCP has to create a client and call Connect() to specify the remote machine address before sending or receiving data.
+```csharp
+using(var xClient = new TcpClient(4242))
+{
+    xClient.Connect(new Address(192, 168, 1, 70), 4242);
+
+    /** Send data **/
+    xClient.Send(Encoding.ASCII.GetBytes(message));
+
+    /** Receive data **/
+    var endpoint = new EndPoint(Address.Zero, 0);
+    xClient.Receive(ref endpoint);  //set endpoint to remote machine IP:port
```

Comment:
Why is this value passed in and not just retuned?

Reply:
Because Receive returns a byte array
![image](https://user-images.githubusercontent.com/18724279/117310897-0227a200-ae84-11eb-9669-bfc3f23db434.png)
My final goal is to plug System.Net but for now all I can do is try not to stray too far from the .NET classes (this is the official Microsoft documentation for UdpClientt)

Reply:
Good point